saltlint/rules/NewLineAtTheEndRule.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Dawid Malinowski <dawidmalina@gmail.com>
+
+from saltlint.linter import SaltLintRule
+
+class NewLineAtTheEndRule(SaltLintRule):
+    id = '216'
+    shortdesc = 'The POSIX standard requires the last line to be a new line character.'
+    description = 'The POSIX standard requires the last line to be a new line character.'
+    severity = 'LOW'
+    tags = ['formatting']
+    version_added = 'develop'
+
+    def matchtext(self, file, text):
+        lines = text.splitlines()
+        results = []
+        if lines and '' != lines[-1]:
```

Comment:
Pylint seems to [want things differently](https://travis-ci.org/warpnet/salt-lint/jobs/615611549?utm_medium=notification&utm_source=github_status):

```suggestion
        if lines and lines[-1] != '':
```

Reply:
Fixed by @dawidmalina in bbca3ec.